mne/channels/montage.py

```diff
@@ -348,6 +347,40 @@ def get_positions(self):
         )
         return positions
 
+    def add_estimated_fiducials(self, subject, subjects_dir):
+        """Estimate fiducials based on FreeSurfer ``fsaverage`` subject.
+
+        Parameters
+        ----------
+        %(subject)s
+        %(subjects_dir)s
+
+        Notes
+        -----
+        Since MNE uses the FIF data structure, it relies on the ``head``
+        coordinate frame. Any coordinate frame can be transformed
+        to ``head`` if the fiducials (i.e. LPA, RPA and Nasion) are
+        defined. One can use this function to estimate those fiducials
+        and then use ``montage.get_native_head_t()`` to get the
+        head <-> MRI transform.
+        """
+        from mne.coreg import get_mni_fiducials
+
```

Comment:
First step of the function should be to check the coord frame. If it's not MRI, raise an error. The description above should make it clear this will only work for montages in MRI (FreeSurfer surface RAS) coordinates.